homeassistant/components/coronavirus/sensor.py

```diff
@@ -62,11 +62,6 @@ def icon(self):
         """Return the icon."""
         return SENSORS[self.info_type]
 
-    @property
```

Comment:
Removing this causes the frontend to stop displaying nice graphs showing inclines and declines in the data.  If you don't think `people` is a valid unit to go by then I think you should suggest a different unit of measurement.  Maybe call it `cases` but then when it comes to `deaths` using `cases` doesn't really make sense.  IMO people are really whats being recorded here.  I mean they are not including data from other species right?